### ISSUE TITLE
Biased anomalies due to non zero SWE in summer 

### ISSUE THREAD

**Issue by sgascoin** [sgascoin/snow-in-gmba#1]
Annual SWE minimum is well above zero in some high latitude mountain polygons, typically in glacierized regions. This multiyear snow accumulation creates a bias in the anomaly calculation. We could fix this by subtracting the mean SWE value in September (although this baseline is not constant as shown below)

<img width="454" alt="image" src="https://github.com/sgascoin/snow-in-gmba/assets/29677722/4ca58a38-7bf2-4b54-b18f-6a25778e72e0">

**Comment by mitmat** [comment]
Yeah, I guess the easiest way would be to subtract Sep SWE. Though, there are two options:

1. For each year x Apr 01 subtract the value Sep 01 year (x-1)
2. For all years, subtract the climatological mean Sep 01 over 1991-2020

I guess 1. is more noisy, but more accurate - and reflect more the interannual variability. It would also give you better the seasonal accumulation, like around year 2003-4, where the accumulated SWE in one season needs multiple years to deplete. 2. would be more robust, but less related to year-to-year SWE accumulation.

**Comment by sgascoin** [comment]
Another thing to consider: it seems that ice-covered cells have a constant SWE set to 10 m over the entire ERA5-Land record (1950-present). Minimum monthly SWE value is 10m. We may want to exclude those cells? 
![Era5Land-minSwe](https://github.com/sgascoin/snow-in-gmba/assets/29677722/678572aa-49c2-4a56-91f6-02bcb60e5930)

**Comment by mitmat** [comment]
In analysis of regional climate model SWE data, these areas are usually discarded. Maybe we should do it, too.

Otherwise, we probably should add a disclaimer about the reliability of ERA5-Land SWE in general? I guess it is only a rough approximation because of the simplicity of the snow module, but I have not actualy read evaluation papers...

**Comment by sgascoin** [comment]
we have made [an evaluation over the Tuolumne river basin](https://egusphere.copernicus.org/preprints/2024/egusphere-2024-791/), it works surprisingly well .. (PS. the colors are difficult to distinguish, it should be fixed in the revision)
<img width="761" alt="image" src="https://github.com/sgascoin/snow-in-gmba/assets/29677722/13a0f93f-04aa-40c8-80b3-711c1abce548">

**Comment by sgascoin** [comment]
There are also the [ongoing studies in the scope of ESA CCi](https://egusphere.copernicus.org/preprints/2024/egusphere-2023-3014/) "The top performing products across the range of tests performed are ERA5-Land followed by the Crocus snow model".  But I agree that a dedicated evaluation would be useful, especially in mountain regions! For instance, the latter study does not cover the Alps.. You have the data :)

**Comment by mitmat** [comment]
Wow, that looks very impressive for the Toulumne. Would be great to see how it works in other regions of the world, maybe we can do something nice after we collected the data from the JB... so not only Alps ;)

I will add the references to the dash "About" page.
